### PR TITLE
Change ui stack

### PR DESCRIPTION
**Pull request by mtalstykh**

### CODE REVIEW

**Comment by skateray17** [comment]
Fix conflicts pls

File: README.md [hunk 1]
```diff
@@ -1,40 +1,70 @@
...
-
-### Run your unit tests
-```
-npm run test:unit
-
-```
-Changes for tests
-### Customize configuration
-See [Configuration Reference](https://cli.vuejs.org/config/).
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```
Comment (skateray17) [inline README.md]: Seems, we should remove this line

**Review by skateray17 — CHANGES_REQUESTED** [review]

**Comment by skateray17** [comment]
Nice

**Comment by mtalstykh** [comment]
> Nice

Wow, you are so fast!

**Review by skateray17 — APPROVED** [review]

**Pull request merged**